test/functional/terminal/scrollback_spec.lua

```diff
@@ -416,7 +438,11 @@ describe("'scrollback' option", function()
     -- Terminal job data is received asynchronously, may happen before the
     -- 'scrollback' option is synchronized with the internal sb_buffer.
     command('sleep 100m')
-    feed_data('for i in $(seq 1 40); do echo "line$i"; done\n')
+    if iswin() then
+      feed_data('for($i=1;$i -le 40;$i++){Write-Host \"line$i\"}\r')
```

Comment:
Got [13, 96, 114].
Get the char codes of carriage return, shell-specific escape character(s), and character(s) following them to resolve character combinations, such as `` `n ``, for terminal buffer.

Unrelated but, is something similar done to cmd.exe for newlines?